style.css

```diff
@@ -512,3 +520,25 @@ form textarea {
     display: block;
   }
 }
+
+/* Special classes */
+
+.hidden {
+  display: none;
+}
+
+.modal {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  z-index: 100;
+  background-color: #6070ff8e;
+}
```

Comment:
- [ ] Please make sure the background color matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=4DHMTBKEmxhBQW4R-0)  design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/208695827-43d6e86a-4a22-4b4e-b3e0-1d8571102bfa.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/208690788-a80ae635-aa21-4347-80b3-a5a7e56297fa.png)